Misc/NEWS.d/next/C API/2018-12-02-22-39-41.bpo-35381.HjI3j_.rst

```diff
@@ -0,0 +1 @@
+Convert posixmodule.c statically allocated types ``DirEntryType`` and ``ScandirIteratorType`` to heap-allocated  types.
```

Comment:
I think this change does not need a news entry. This is just internal refactoring, it does not have anything to users.